PublicationsRouter/v2/sword-out/pubrouter-xml/pubrouter.xsd

```diff
@@ -0,0 +1,285 @@
+                <xs:annotation>
+                    <xs:documentation xml:lang="en">
+                        Type of the contributor, such as "editor".
+                    </xs:documentation>
+                </xs:annotation>
+            </xs:element>
+            <xs:element name="id" minOccurs="0" maxOccurs="unbounded">
+                <xs:annotation>
+                    <xs:documentation xml:lang="en">
+                        Identifier for this contributor, such as ORCID.
+                    </xs:documentation>
+                </xs:annotation>
+                <xs:complexType>
+                    <xs:simpleContent>
+                        <xs:extension base="xs:string">
+                            <xs:attribute name="type" type="xs:string" use="required"/>
+                        </xs:extension>
+                    </xs:simpleContent>
+                </xs:complexType>
+            </xs:element>
+            <xs:element name="email" minOccurs="0" maxOccurs="unbounded" type="xs:string">
+                <xs:annotation>
+                    <xs:documentation xml:lang="en">
+                        Email address for this contributor.
+                    </xs:documentation>
+                </xs:annotation>
+            </xs:element>
+            <xs:element name="surname" minOccurs="0" type="xs:string">
+                <xs:annotation>
+                    <xs:documentation xml:lang="en">
+                        Surname of this contributor.
+                    </xs:documentation>
+                </xs:annotation>
+            </xs:element>
+            <xs:element name="firstnames" minOccurs="0" type="xs:string">
+                <xs:annotation>
+                    <xs:documentation xml:lang="en">
+                        Comma separated first names for this contributor.
+                    </xs:documentation>
+                </xs:annotation>
+            </xs:element>
+            <xs:element name="suffix" minOccurs="0" type="xs:string">
+                <xs:annotation>
+                    <xs:documentation xml:lang="en">
+                        Name suffix for this contributor
+                    </xs:documentation>
+                </xs:annotation>
+            </xs:element>
+            <xs:element name="org_name" minOccurs="0" type="xs:string">
+                <xs:annotation>
+                    <xs:documentation xml:lang="en">
+                        Name of the organisation that authored this publication.
+                    </xs:documentation>
+                </xs:annotation>
+            </xs:element>
+        </xs:all>
+    </xs:complexType>
+    
+    <xs:element name="entry">
+        <xs:annotation>
+            <xs:documentation xml:lang="en">
+                Schema made for JISC Publications Router
+                
+                Created 2017-08
+                
+                This schema is to provide maximum meta-data information for Publications Router to ingest.
```

Comment:
Can this annotation be moved as close to top of XSD as possible.

NB. Not for ingest.

Schema is to expose maximum meta-data in XML format for recipients, such as repositories, to receive and ingest.